### DISCUSSION TITLE
React Developer

### DISCUSSION THREAD

**Discussion by erffon** [WeAreMahsaAmini/Core#196] [Introductions]
Hi guys 👋
my name is Mohammad Erfan Ranjbaran
I am a Junior front-end developer that develops website using **React JS** . It would be an honor for me to participate in this project.

contact info:
[Linkedin  ](https://www.linkedin.com/in/erff-on/)|  [Instagram  ](https://www.instagram.com/erff_on/)|  [Telegram](https://t.me/erff_on)

**Comment by ehsaaanes** [comment]
Hi
I'm looking for people who can help creating an app for donation. 
It would be a great help if you check the idea here #163 and write a comment or let me know what you think about it.
So grateful for your time and attention

**Reply by erffon** [reply]
سلام 
پروژتون رو دیدم و یه ایده ای به ذهنم رسید که بنظرم میتونه اساسی تر و اثربخش تر باشه . توی این لینک مشاهده بفرمایید #266

**Comment by arzybekb** [comment]
Good luck brother!
